caluma/data_source/schema.py

```diff
@@ -0,0 +1,35 @@
+from graphene import Connection, ConnectionField, String
+from graphene.types import ObjectType
+
+from .data_source_handlers import get_data_source_data, get_data_sources
+
+
+class DataSource(ObjectType):
+    info = String()
```

Comment:
Since `name` is technical, should we use this field for display in the form builder UI? This might need to be multilingual then.

Reply:
Done. Also the labels for the options is now multilingual.